.github/workflows/velox_be.yml

```diff
@@ -545,7 +544,7 @@ jobs:
       - name: TPC-H SF1.0 && TPC-DS SF1.0 Parquet local spark3.2 (centos 8)
         run: |
           docker run --rm --init --privileged --ulimit nofile=65536:65536 --ulimit core=-1 --security-opt seccomp=unconfined \
-          -v $PWD:/opt/gluten --name static-build-test-$GITHUB_RUN_ID-tpc -e NUM_THREADS=30 $LEGACY_DOCKER_REGISTRY/gluten-dev/centos:8 \
+          -v $PWD:/opt/gluten --name static-build-test-$GITHUB_RUN_ID-tpc -e NUM_THREADS=30 $DOCKER_PULL_REGISTRY/gluten-te/gluten-build-centos:8 \
```

Comment:
As it's static build, let's do the verification on a pure centos-8 obtained from public hub, without any dedicatedly installed dependencies. Ditto for the below ubuntu20/22 test.

Reply:
If we use a pure centos-8 image，
Some dependencies will be missing.

For example, java will be missing： https://github.com/oap-project/gluten/actions/runs/7558217894/job/20579225893?pr=4427#step:6:23

Reply:
We may need a pure docker with only minimum required dependencies installed. Let's firstly find what are required.

Reply:
I have made a new commit to use pure docker images